webpack.config.js

```diff
@@ -5,6 +5,11 @@ Encore
     .setPublicPath('/build')
     .cleanupOutputBeforeBuild()
     .autoProvidejQuery()
+    .autoProvideVariables({
+        "window.jQuery": "jquery",
```

Comment:
I made a PR on [webpack-encore](https://github.com/symfony/webpack-encore/pull/10) to provide this automatically